### PR TITLE
Use podDesc from kubelet/util/format

### PR DESCRIPTION
**Pull request by AlexeyPerevalov**
This commit just removes code duplication.

/kind cleanup

### CODE REVIEW

File: pkg/kubelet/dockershim/network/cni/cni.go [hunk 1]
```diff
@@ -30,12 +30,14 @@ import (
 
 	"github.com/containernetworking/cni/libcni"
 	cnitypes "github.com/containernetworking/cni/pkg/types"
+	machinerytypes "k8s.io/apimachinery/pkg/types"
```
Comment (SergeyKanzhelev) [inline pkg/kubelet/dockershim/network/cni/cni.go:33]: this is a new alias for this import =). Consider using one of existing ones:

```
kubetypes "k8s.io/apimachinery/pkg/types"
k8stypes "k8s.io/apimachinery/pkg/types"
types "k8s.io/apimachinery/pkg/types"
commontypes "k8s.io/apimachinery/pkg/types"
```

also it is used so often so maybe worth placing into `hack/.import-aliases`

File: pkg/kubelet/dockershim/network/cni/cni.go [hunk 2]
```diff
@@ -347,18 +349,14 @@ func (plugin *cniNetworkPlugin) TearDownPod(namespace string, name string, id ku
 	return plugin.deleteFromNetwork(cniTimeoutCtx, plugin.getDefaultNetwork(), name, namespace, id, netnsPath, nil)
 }
 
-func podDesc(namespace, name string, id kubecontainer.ContainerID) string {
-	return fmt.Sprintf("%s_%s/%s", namespace, name, id.ID)
```
Comment (SergeyKanzhelev) [inline pkg/kubelet/dockershim/network/cni/cni.go:351]: `format.PodDesc` is using `"%s_%s(%s)"` as a format. Here the `/` is used. It may be intentional.

Comment (SergeyKanzhelev) [reply pkg/kubelet/dockershim/network/cni/cni.go:351]: Comment to when this method was introduced (for history): https://github.com/kubernetes/kubernetes/pull/69159#discussion_r221402772

Comment (SergeyKanzhelev) [reply pkg/kubelet/dockershim/network/cni/cni.go:351]: and method existed long before this: https://github.com/kubernetes/kubernetes/pull/36020/files

**Comment by SergeyKanzhelev** [comment]
I'm not sure what we gain out of this refactoring. Also it changes output (insignificantly). I'd suggest to hold this PR for better description on why it's needed.

/hold

**Comment by AlexeyPerevalov** [comment]
> I'm not sure what we gain out of this refactoring. Also it changes output (insignificantly). I'd suggest to hold this PR for better description on why it's needed.
> 
> /hold

Are you afraid it could break log analyzers?

**Comment by bboreham** [comment]
@SergeyKanzhelev has a valid point.  I don't think it's worth the type-cast to save 3 lines.

However I am now uneasy that people could be confused by the similar-looking rendering of pod UID or container ID.

**Comment by SergeyKanzhelev** [comment]
> > I'm not sure what we gain out of this refactoring. Also it changes output (insignificantly). I'd suggest to hold this PR for better description on why it's needed.
> > /hold
> 
> Are you afraid it could break log analyzers?

This too. But mostly being cautious. With the different format there is at least some difference between pod ID and container ID. Maybe @kubernetes/sig-instrumentation-pr-reviews may suggest something or give some historical perspective.

**Comment by SergeyKanzhelev** [comment]
/close

please re-open if there is a desire to pursue it further

**Pull request closed without merging**